script/build-release

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+echo "$KEYSTORE_PROPERTIES" > keystore.properties
+echo -n "$STORE_FILE" | base64 --decode > ./app/vinyl.jks
```

Comment:
Considering the previous comment (about retaining the "unsecured" keys), I'd move this `vinyl.jks` file to the same `scripts` folder.

Do you plan to share this Play Store signing keys (via a separate secured channel)? I dont know if there is a way in Github to store the file in a protected "vault", not wide open. This will make the release process fully automated, without your manual intervention or relying on your environment.

Reply:
yes, this is actually what is already done :) if you look at the [project secrets here](https://github.com/VinylMusicPlayer/VinylMusicPlayer/settings/secrets/actions)  you will see the `STORE_FILE` variable that contains the base64 key used to sign the release in the Play Store. So what you are suggesting is already implemented, and the 1.5.3 release present in the Play Store was built automatically from the tag using this process.

This is why I think we can use the same key for CI builds as well given it is already protected.

Reply:
Okay, one can then switch between Play Store vetted version and the CI builds without losing the settings/play history. This is nice - though I still need to reset everything since I've been using the unsecure-signed version for a while now :-)

Reply:
Hmm, looks like that I need to have the keystore/vinyl.jks file locally to be able to build and debug - before pushing to github.
It's impossible to extract the secrets from github to rebuild that file - I guess this is intentionally done like that to protect the secrets.

I will restore the old unsecured keys, and change the pipeline so that all contributors wont face this  issue. @AdrienPoupa unless you have a better solution?

Reply:
This is odd, when I was building from Android Studio (using the Play button) never used the CI key as far as I remember. I only used the prod one when I went to generate a signed release.

If you want to restore the unsigned key please leave the Play Store and CI flavors as they are right now and create a new flavor with your key.

Reply:
It works on your end since you have the `keystore.properties` file locally (which contains the secured PS keys - supposedly). This file is not gitted - on purpose - my local build fails since the file is not present.

Please have a look at the proposed PR https://github.com/VinylMusicPlayer/VinylMusicPlayer/pull/791 - where I created a copy of `keystore.properties` for all contributor to use, pointing to the unsecured keys.

The CI/release builds on Github are unaffected since I dont change the workflow. They should build the file from the injected environment variabes, coming from the GH secrets (cf the `build-release` script)